cmd/logs.go

```diff
@@ -3,11 +3,14 @@ package cmd
 import (
 		"fmt"
 		"strings"
+	  "time"
```

Comment:
so I did run go fmt. Nothing was changed. If you actually pull this code down, I bet it does not look like this. I think github may be doing something funny with go files. Not sure though.